rest-api/tools/csv_importer.py

```diff
@@ -64,6 +64,9 @@ def run(self, filename, dry_run):
             self._insert_entity(entity, existing_map, session, dry_run)
             new_count += 1
 
+    if self.errors:
+      for err in self.errors:
+        logging.warn(err)
     logging.info('Done importing %ss%s: %d skipped, %d new, % d updated, %d not changed',
```

Comment:
Add the number of errors in this summary message.

Reply:
done